calendar.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2017 Keegan Joseph Brophy
```

Comment:
You should only assert copyright over things that you have written. In this case, you've incorporated work that I did previously... but which I failed to license to you (oops!). The correct course of action here is for me to add a copyright notice and license to https://github.com/trombonehero/memorial-calendar/blob/master/calendar.py and then for you to pick up that version. If you've substantially modified this file, you can then add your copyright alongside mine.